modules/ossm-rn-deprecated-features.adoc

```diff
@@ -15,7 +15,7 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 == Deprecated features {ProductName} 2.0
 
-The Mixer component is deprecated and will be removed in a future release.  While using Mixer for implementing extensions is still supported in release 2.0, you should be migrating your extensions to the new link:https://istio.io/latest/blog/2020/wasm-announce/[WebAssembly] mechanism.
+The Mixer component was deprecated in release 2.0 and was removed in release 2.1.  While using Mixer for implementing extensions was still supported in release 2.0, extensions should have been migrated to the new link:https://istio.io/latest/blog/2020/wasm-announce/[WebAssembly] mechanism.
```

Comment:
@JStickler line 18, we haven't released 2.1 yet. So should we replace "was removed in release 2.1" with "will be removed in release 2.1" ?

Reply:
Good catch, I was getting ahead of myself.  Fixed.